### ISSUE TITLE
Wrong image

### ISSUE THREAD

**Issue by pstaykov** [huggingface/course#1215]
The image showcasing accuracy over training steps in chapters/en/chapter3/5.mdx actually shows a duplicate loss over steps graph probably showing overfitting (climbing validation loss) but not the intended accuracy graph.
